README.md

```diff
@@ -601,6 +601,7 @@ _Icecast clients_
 * [ices2](http://www.icecast.org/ices.php) - Ogg Vorbis streaming source for Icecast 2 ([◼](https://packages.debian.org/sid/ices2))
 * [liquidsoap](http://savonet.sourceforge.net/) - audio streaming language ([◼](https://packages.debian.org/sid/liquidsoap))
 * [oggfwd](http://v2v.cc/~j/oggfwd/) - basic icecast source client for audio and video streaming ([◼](https://packages.debian.org/sid/oggfwd))
+* [DeeFuzzer](https://github.com/yomguy/DeeFuzzer) - light and instant streaming client for audio and video written in Python
```

Comment:
@yomguy I will take care of fixing alphabetical ordering since I have other minor cleanup tasks on the way. Please consider "fixing" installation instructions as recommended above if you find the time. Thanks